### DISCUSSION TITLE
contribute a clj-kondo hook?

### DISCUSSION THREAD

**Discussion by rgm** [mhuebert/inside-out#3] [Ideas]
Hi, I'm loving inside-out so far. I got a start on a clj-kondo hook that I'd like to contribute:

https://gist.github.com/rgm/3e18528dab5b878afa58f0a87f73aa4e

Should I send this as PR? It's my first try at writing a hook, so it could probably be better.

**Comment by anthonygalea** [comment]
Thanks for sharing this @rgm, also loving inside-out here, unfortunately I don't have a lot of experience writing clj-kondo hooks, just wanted to point out a (more blunt) alternative:

```clj
{:linters
 {:unresolved-symbol
  {:exclude-patterns ["^\\?"]}}}
```
